lighthouse-core/scripts/compare-timings.js

```diff
@@ -188,22 +238,24 @@ function filter(results) {
 }
 
 /**
- * @param {number=} value
+ * @param {number|string|undefined} value
  * @return {value is number}
  */
 function exists(value) {
   return typeof value !== 'undefined';
 }
 
 function summarize() {
-  const results = aggregateResults(argv.name);
-  filter(results);
-  print(results);
+  for (const resultType of ['timings', 'metrics']) { 
```

Comment:
not a fan of printing both of these unconditionally. seems you'd only want to look at timings or metrics, not both.

Reply:
k done and documented.